gm4_storage_drawers/data/storage_drawers/functions/all/destroy.mcfunction

```diff
@@ -0,0 +1,14 @@
+#run from pulse_check
+#@s = as and at drawer with base block destroyed
+
+execute if score @s gm4_sd_amount matches 65.. run tag @s add gm4_sd_filled
+execute if entity @s[tag=gm4_sd_filled] run setblock ^ ^ ^-1 minecraft:white_bed keep
+execute if entity @s[tag=gm4_sd_filled] run title @p[distance=..3] actionbar "Drawer needs 64 items or less before you destroy it!"
```

Comment:
This is not really user friendly. You could use recursion to drop multiple item stacks.